lxd/config/map.go

```diff
@@ -56,7 +56,13 @@ func (m *Map) Change(changes map[string]any) (map[string]string, error) {
 			continue
 		}
 
-		values[name] = change.(string)
+		value, ok := change.(string)
+		if !ok {
+			errors.add(name, nil, fmt.Sprintf("failed to cast value %q to string", change))
```

Comment:
Upper case F as errors should start with capital please

Reply:
I was following the style of the other messages in this file/package but then I'll modify the others too.

Reply:
ah ok i didnt see those in the diff.

Reply:
It requires a few more changes also in the respective test suites as they are checking for the specific error message. Would you rather have this in another PR? Otherwise I'll add it here.

Reply:
lets leave it as is for now then